.github/workflows/build.yml

```diff
@@ -6,23 +6,23 @@ on:
   pull_request:
 
 jobs:
-  ansible-lint:
-    runs-on: ubuntu-20.04
+  #ansible-lint:
+  #  runs-on: ubuntu-20.04
 
-    steps:
-      - uses: actions/checkout@v3
+  #  steps:
+  #    - uses: actions/checkout@v3
 
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.10.15'
+  #    - uses: actions/setup-python@v4
+  #      with:
+  #        python-version: '3.10.15'
 
-      - name: Set up Ansible
-        run: |
-          pip install -r requirements.txt
-          bin/setup
+  #    - name: Set up Ansible
+  #      run: |
+  #        pip install -r requirements.txt
+  #        bin/setup
 
-      - name: Ansible Lint
-        run: ansible-lint playbooks/*.yml --exclude community
+  #    - name: Ansible Lint
+  #      run: ansible-lint playbooks/*.yml --exclude community
```

Comment:
I wonder if there's a way we could configure it to always skip. Then it would still show in GitHub as an action that doesn't run (and hence remind there's something broken to fix one day)